src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+        boolean exitFlag = false;
+        int index = 0;
+        // Problem will come if there are more than 100 tasks
+        Task[] list = new Task[100];
+        printWelcomeMessage();
+        while (!exitFlag) {
+            // Read User input
+            Scanner in = new Scanner(System.in);
+            String line = in.nextLine();
+            // process user input and decide what operation to use
+            String operation = extractOperationType(line);
+            String printStatement = null;
+
+            switch (operation){
+                case "bye":
```

Comment:
Switch statements: as per the coding standards, switch statements do no require indentations